nest_elephant_tvb/translation/FatEndPoint.py

```diff
@@ -0,0 +1,196 @@
+from mpi4py import MPI
```

Comment:
As a new file it should have license information at the start

 Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
"Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements; and to You under the Apache License, Version 2.0. "

Reply:
Resolved this. Accidently create d new conversation instead of replying.